packages/lib/src/vis/heatmap/HeatmapMesh.tsx

```diff
@@ -22,6 +22,7 @@ interface Props {
   magFilter?: TextureFilter;
   alphaValues?: NdArray<TextureSafeTypedArray | Uint16Array>; // uint16 values are treated as half floats
   alphaDomain?: Domain;
+  size?: Size;
```

Comment:
I wasn't too sure what to add as props: `size`, `width` and `height`, `geometrySize`... ?
I also hesitated to `extend VisMeshProps`, but I don't need it and `scale` is used by `HeatmapMesh`.

Reply:
Let's stick with this for now, I reckon, it's nice and simple. We can refactor later.